ksql-engine/src/main/java/io/confluent/ksql/schema/registry/DefaultSchemaRegistryClient.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.schema.registry;
+
+import io.confluent.kafka.schemaregistry.ParsedSchema;
+import io.confluent.kafka.schemaregistry.client.SchemaMetadata;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.rest.entities.SchemaReference;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.apache.kafka.common.config.ConfigException;
+
+/** 
+ * Implements the SchemaRegistryClient interface. Used as default when the Schema Registry URL isn't
+ * specified in {@link io.confluent.ksql.util.KsqlConfig}
+ */
+public class DefaultSchemaRegistryClient implements SchemaRegistryClient {
+
+  private static final ConfigException CONFIG_EXCEPTION =
+      new ConfigException("Schema Registry URL not set");
```

Comment:
Maybe make this more informative? `"KSQL is not configured to use a schema registry. To enable it, please set ksql.schema.registry.url"`

Reply:
done